src/js/views/patients/patient/patient.scss

```diff
@@ -140,14 +140,22 @@
 }
 
 .patient__list-meta {
-  width: 510px;
+  width: 562px;
```

Comment:
I'm going to play around with these widths a little more on the patient flow page. When this meta section is add it's maximum potential width, things look a little funky to me.

Reply:
could be a min-width?

Reply:
Min-width breaks the layout on smaller width browser windows. 

I think what we have is fine right now, but I'll make a note on the card for Mark so he can take a look when this is up on QA. And see if it looks good to him and we can make adjustments if needed.